Src/FluentAssertions.Dotnet/Dotnet.csproj

```diff
@@ -29,11 +29,11 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>..\..\Artifacts\Lib\dotnet\</OutputPath>
```

Comment:
So ideally I just rename those 'dot net' projects to something like 'netstandard' right?